tests/general/test_namespace.py

```diff
@@ -189,8 +188,7 @@ def test_deletion_no_throttle(db, patch_requests_no_throttle):
     db.session.commit()
 
     to_delete = get_accounts_to_delete(0)
-    greenlet = gevent.spawn(batch_delete_namespaces, to_delete, throttle=True)
-    greenlet.join()
```

Comment:
Those just didn't make sense at all. This spawns a greenlet with a function and then immediately waits for it to end. Logically it's exactly the same as running the function in the same greenlet/thread.